program/src/balance.rs

```diff
@@ -31,28 +31,31 @@ pub fn get_target_balance(
     );
 
     let total_delegated_lamports: token::Result<Lamports> = validators
-        .iter_entries()
+        .iter_active()
```

Comment:
This needs to iterate all validators, not just the active ones. Right after if we flip the switch, the validator will still have active stake that is part of the Solido balance, it doesn’t disappear.